lib/portage/gpkg.py

```diff
@@ -764,10 +764,10 @@ class gpkg:
     https://www.gentoo.org/glep/glep-0078.html
     """
 
-    def __init__(self, settings, base_name=None, gpkg_file=None):
+    def __init__(self, settings, basename=None, gpkg_file=None):
         """
         gpkg class handle all gpkg operations for one package.
-        base_name is the package basename.
+        basename is the package basename.
```

Comment:
It's a bit weird to call it "basename" when it contains category name as well.

Reply:
It suppose to be basename, but also support pass cpv to it, or the caller need to split it themselves.